webseed/request_test.go

```diff
@@ -23,7 +23,7 @@ func TestTrailingPath(t *testing.T) {
 }
 
 func TestTrailingPathForEmptyInfoName(t *testing.T) {
-	qt.Check(t, trailingPath("", []string{`ノ┬─┬ノ ︵ ( \o°o)\`}), qt.Equals, "%E3%83%8E%E2%94%AC%E2%94%80%E2%94%AC%E3%83%8E+%EF%B8%B5+%28+%5Co%C2%B0o%29%5C")
-	qt.Check(t, trailingPath("", []string{"hello", "world"}), qt.Equals, "hello/world")
-	qt.Check(t, trailingPath("war", []string{"and", "peace"}), qt.Equals, "war/and/peace")
+	qt.Check(t, trailingPath("", []string{`ノ┬─┬ノ ︵ ( \o°o)\`}, url.QueryEscape), qt.Equals, "%E3%83%8E%E2%94%AC%E2%94%80%E2%94%AC%E3%83%8E+%EF%B8%B5+%28+%5Co%C2%B0o%29%5C")
+	qt.Check(t, trailingPath("", []string{"hello", "world"}, url.QueryEscape), qt.Equals, "hello/world")
+	qt.Check(t, trailingPath("war", []string{"and", "peace"}, url.QueryEscape), qt.Equals, "war/and/peace")
 }
```

Comment:
Another test for example for an encoding that handles commas or spaces differently could be useful here, at least just to show that the default query escaping isn't the only one that works.

Reply:
Done https://github.com/afjoseph/torrent/commit/a79c3bd5d30d99fb33ab7c5334c7d017884d4403